grafana-plugin/src/containers/AlertRules/AlertRules.tsx

```diff
@@ -155,20 +155,100 @@ class AlertRules extends React.Component<AlertRulesProps, AlertRulesState> {
       <>
         <div className={cx('root')}>
           <Block className={cx('headerBlock')}>
-            <div className={cx('header')}>
-              <Text.Title level={4}>
-                <HorizontalGroup>
-                  Escalate
-                  <div className={cx('verbal-name')}>{parseEmojis(alertReceiveChannel?.verbal_name || '')}</div>
-                  <Tooltip placement="top" content="Edit name">
+            <div className={cx('integration__heading-container')}>
+              <div className={cx('integration__heading-container-left')}>
+                <Text.Title level={4}>
+                  <div className={cx('integration__heading-text')}>
+                    <div className={cx('verbal-name')}>{parseEmojis(alertReceiveChannel?.verbal_name || '')}</div>
+                    <Tooltip placement="top" content="Edit name">
+                      <span>aaadsa dasd</span>
```

Comment:
`aaadsa dasd` ?

Reply:
Oh... that was a stupid mistake. Probably left from testing, I didn't notice it - good catch :thumbsup: